USER_GUIDE.md

```diff
@@ -25,23 +38,11 @@ class MyOpenSearchClass
     public function __construct()
     {
         // Simple Setup
-        $this->client = OpenSearch\ClientBuilder::fromConfig([
-            'hosts' => [
-                'https://localhost:9200'
-            ],
-            'retries' => 2,
-            'handler' => OpenSearch\ClientBuilder::multiHandler()
+        $client = (new \OpenSearch\GuzzleClientFactory())->create([
+            'base_uri' => 'https://localhost:9200',
+            'auth' => ['admin', getenv('OPENSEARCH_PASSWORD')],
```

Comment:
should we maybe referrer  to `$_SERVER['OPENSEARCH_PASSWORD']`. getenv is considered bad as it is not thread-safe.

Reply:
@shyim that would only matter if you are changing the password though right?